### ISSUE TITLE
I am sharing the error code by the system that occurred while applying the icon pack

### ISSUE THREAD

**Issue by cehnemdark** [KieronQuinn/PixelLauncherMods#29]
https://paste.crdroid.net/cnKAtj

if it's needed, icon pack app:
https://play.google.com/store/apps/details?id=com.rainystudio.oreo8iconpack

**Comment by KieronQuinn** [comment]
```
time: 1649991693704
msg: android.content.res.Resources$NotFoundException: Resource ID #0xa
stacktrace: android.content.res.Resources$NotFoundException: Resource ID #0xa
	at android.content.res.ResourcesImpl.getValue(ResourcesImpl.java:225)
	at android.content.res.ResourcesImpl.openRawResource(ResourcesImpl.java:340)
	at android.content.res.Resources.openRawResource(Resources.java:1378)
	at android.content.res.Resources.openRawResource(Resources.java:1322)
	at com.kieronquinn.app.pixellaunchermods.utils.extensions.Extensions_ResourcesKt.rawResourceContains(Extensions+Resources.kt:15)
	at com.kieronquinn.app.pixellaunchermods.utils.extensions.Extensions_ResourcesKt.isAdaptiveIcon(Extensions+Resources.kt:11)
	at com.kieronquinn.app.pixellaunchermods.repositories.IconPackRepositoryImpl.parseAppFilter(IconPackRepository.kt:306)
	at com.kieronquinn.app.pixellaunchermods.repositories.IconPackRepositoryImpl.getAppFilter(IconPackRepository.kt:226)
	at com.kieronquinn.app.pixellaunchermods.repositories.IconPackRepositoryImpl.getAppFilter$default(IconPackRepository.kt:200)
	at com.kieronquinn.app.pixellaunchermods.repositories.IconPackRepositoryImpl$getAllComponentIcons$2.invokeSuspend(IconPackRepository.kt:173)
	at com.kieronquinn.app.pixellaunchermods.repositories.IconPackRepositoryImpl$getAllComponentIcons$2.invoke(Unknown Source:8)
	at com.kieronquinn.app.pixellaunchermods.repositories.IconPackRepositoryImpl$getAllComponentIcons$2.invoke(Unknown Source:4)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:165)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source:1)
	at com.kieronquinn.app.pixellaunchermods.repositories.IconPackRepositoryImpl.getAllComponentIcons(IconPackRepository.kt:172)
	at com.kieronquinn.app.pixellaunchermods.repositories.RemoteAppsRepositoryImpl$autoApplyIconPacks$1.invokeSuspend(RemoteAppsRepository.kt:454)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.UndispatchedCoroutine.afterResume(CoroutineContext.kt:147)
	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.internal.LimitedDispatcher.run(LimitedDispatcher.kt:39)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:95)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

**Comment by KieronQuinn** [comment]
Fixed in 2.2
